app/login/route.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  actions: {
+
+    login() {
+
+      var controller = this.get('controller');
+      var email = controller.get('userEmail');
+      var password = controller.get('userPassword');
+
+      this.get('session').open('firebase', {
+        provider: 'password',
+        email: email,
```

Comment:
Because of [enhanced objects literals](https://babeljs.io/learn-es2015/#ecmascript-2015-features-enhanced-object-literals) in ES6, you can just have:
```js
{
    email,
}
```
That would yield: 
```js
{
    email: email,
}
```